tests/subsys/dsp/basicmath/src/f16.c

```diff
@@ -9,7 +9,7 @@
 #include <zephyr/kernel.h>
 #include <stdlib.h>
 #include <arm_math_f16.h>
-#include "../../common/test_common.h"
+#include "common/test_common.h"
 
 #include "f16.pat"
```

Comment:
This file still contains `arm_` prefixed function (i.e. CMSIS-DSP) tests in spite of being part of a **ZDSP** test.

There needs to exist logical separation between CMSIS-DSP and ZDSP and it does not make sense for a ZDSP test to directly test CMSIS-DSP functions.

Reply:
> Note that f16 is not updated since it's much more architecture specific and did not become a part of zdsp.

`_Float16` is available as part of ISO/IEC TS 18661-3, which is an extension to the ISO/IEC 9899 (i.e. ISO C standard). It would make sense for the ZDSP subsystem to support 16-bit float type.

Reply:
Done